pkg/generator.go

```diff
@@ -752,6 +752,10 @@ func (_m *{{.MockName}}{{.InstantiatedTypeString}}) {{.FunctionName}}({{join .Pa
 {{- else}}
 	{{- .RetVariableName}} := {{.Called}}
 
+	if len({{.RetVariableName}}) == 0 {
+		panic("Missing Return() function for {{.FunctionName}}()")
```

Comment:
Alternatively, I'd say the message should be:

```
"no return value specified for {{.FunctionName}}"
```

Return values can be specified in multiple different ways, not just through `.Return()`.